content/docs/ops/setup/app-health-check/index.md

```diff
@@ -166,7 +115,18 @@ spec by yourself.
 The configuration changes above (by Helm or by the configuration map) effect all Istio app deployments.
 {{< /warning >}}
 
-##### Re-deploy the liveness health check app
+#### Use Annotations on Pod
+
```

Comment:
I think this option should go first.  It is most likely users will use annotation before enable it globally.

Reply:
Maybe we should even focus on document this approach and then in the end mention the Helm flag can turn it on globally as a side note? That makes the doc even simpler.

Reply:
This might suits well in the follow-up improvements.